R/stats-glm-tidiers.R

```diff
@@ -5,13 +5,20 @@
 #' @template param_confint
 #' @template param_exponentiate
 #' @template param_unused_dots
+#' @param conf.method Controls whether stats::confint.default (Wald method)
```

Comment:
Here I would emphasis the choice between estimators/methods rather than the code underlying those methods. i.e. Wald method is more important to know about than `stats::confint.default()`. Additionally, it would be nice to mention what `summary.glm()` does/what you get if you call `summary()` on a `glm` object with default arguments, since that's unclear from the method names themselves.